RelationalAI/TransactionAsyncAbortReasons.cs

```diff
@@ -0,0 +1,31 @@
+﻿/*
+ * Copyright 2022 RelationalAI, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Runtime.Serialization;
+
+namespace RelationalAI
+{
+    public static class TransactionAsyncAbortReason
```

Comment:
The Linter wants this name to match the file name. Either make this name plural or rename the file to be singular. No preference from me; i don't know C# idioms.